ngraph/core/src/op/convert.cpp

```diff
@@ -111,6 +111,8 @@ namespace convert
             break;
             TYPE_CASE(i8)(arg, out);
             break;
+            TYPE_CASE(i16)(arg, out);
```

Comment:
@ilyachur I remember you removed this type from evaluate methods, do we still avoid this type?

Reply:
If we don't have real cases which require it, I propose to remove it. Yes I have removed all redundant data types for nGraph evaluate methods and it helped to decrease a binary size around 1.5 Mb.

Reply:
I've removed this TYPE_CASE call and adjusted the other ops in this PR. As a result of this PR libngraph.so dropped from 7.3MB to 6.4MB on my machine.